Content/posts/hosting-the-swift-package-index.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2021-03-30 12:00
+title: Hosting the Swift Package Index
+description: Making any open-source project sustainable for the long term is challenging in many ways, but with the Swift Package Index we also need significant hardware resources to keep it available.
+---
+
+![Logo images for the Swift Package Index, MacStadium, and Microsoft Azure](/images/hosted-by-macstadium-and-microsoft-azure.png)
+
+Making any open-source project sustainable for the long term is challenging. You need to find time to work on the code, plan out features, respond to feedback, and if the project is not only code but a live web site, it also needs hosting and regular maintenance.
+
+The Swift Package Index has slightly bigger hosting problems than most web sites too. In addition to web and database servers, we need a significant amount of processing power to monitor and analyse constant package releases and run our build machines that deal with the ~30,000 Swift builds per month that process through our [compatibility build system](https://blog.swiftpackageindex.com/posts/launching-language-and-platform-package-compatibility/). 😅
```

Comment:
Just marking the `, too.` here as a reminder as it's not updated yet (perhaps pending the push).